src/screens/ProfileScreen/index.js

```diff
@@ -621,12 +618,12 @@ const ProfileScreen = ({route}) => {
         style={styles.flatlistContainer}
         onScroll={handleScroll}
         ListFooterComponent={isFetchingList ? <ActivityIndicator /> : null}
-        onEndReach={handleOnEndReached}
         initialNumToRender={2}
         maxToRenderPerBatch={2}
         updateCellsBatchingPeriod={10}
         removeClippedSubviews
         windowSize={10}
+        onMomentumScrollEnd={(event) => fetchNextFeeds(event)}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `onEndReach` prop has been replaced with `onMomentumScrollEnd`. Ensure that this change doesn't affect the scrolling behavior and the loading of new feeds.

```diff
- onEndReach={handleOnEndReached}
+ onMomentumScrollEnd={(event) => fetchNextFeeds(event)}
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->